### DISCUSSION TITLE
CommandBase

### DISCUSSION THREAD

**Discussion by quangas** [MarimerLLC/csla#1857] [General]
I've got a legacy project littered with inheritance of CommandBase everywhere. The latest versions of CLSA 5.* from what I have seen requires a type (CommandBase<T>), do we know which version of CLSA dropped the support for this?

**Comment by rockfordlhotka** [comment]
I am not sure I understand the question. The `CommandBase<T>` type remains a valid base class.

```c#
  [Serializable]
  public class MyCommand : CommandBase<MyCommand>
  {

  }
```

**Reply by quangas** [reply]
I found what I was looking for, I was referring to an old library (v4) where < T > was not used. Thanks.
